content/en/blog/_posts/2025-05-22-gateway-api-v1.3/index.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: blog
+title: "Gateway API v1.3.0: Advancements in Request Mirroring, CORS, Gateway Merging, and Retry Budgets"
+date: 2025-05-22T09:00:00-08:00
```

Comment:
Can we hold off until early next week to give folks a bit more time to review this?

Reply:
Changed to May 28.